src/components/cards/MyWallet/MyWallet.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <BalCard noPad :hFull="upToLargeBreakpoint" noShadow>
+    <div class="flex flex-col bg-white w-full">
+      <div class="flex justify-between p-3 shadow-lg">
+        <h6>My Wallet</h6>
+        <div v-if="!isLoadingBalances">{{ etherBalance }} ETH</div>
+        <BalLoadingBlock v-else class="h-8 w-12" />
+      </div>
+      <div class="p-3">
+        <BalLoadingBlock v-if="isLoadingBalances" class="h-8" />
+        <BalAssetSet
+          v-else-if="isWalletReady"
+          @click="setTokenInAddress"
+          :width="230"
+          wrap
+          :size="32"
+          :addresses="tokensWithBalance"
+        ></BalAssetSet>
+        <div v-else class="w-full flex justify-center">
+          <BalBtn size="xs" @click="toggleWalletSelectModal"
+            >Connect your wallet</BalBtn
+          >
```

Comment:
Again, lets get this text into locales and also can we cleanup the schema. I'd also make this a block btn and maybe not as small?
```suggestion
          <BalBtn size="sm" block @click="toggleWalletSelectModal">
              Connect your wallet
          </BalBtn>
```